impeller/entity/inline_pass_context.h

```diff
@@ -18,7 +18,7 @@ class InlinePassContext {
   };
 
   InlinePassContext(std::shared_ptr<Context> context,
-                    RenderTarget render_target,
+                    const RenderTarget& render_target,
```

Comment:
This introduces a copy where one could have been elided before. That is, if the caller had moved in a render target. Now, the const reference necessitates a copy.

Reply:
It doesn't necessitate one: https://godbolt.org/z/vEbY44YMP

Clang can just ignore the move.